db_stress_tool/batched_ops_stress.cc

```diff
@@ -613,6 +613,15 @@ class BatchedOpsStressTest : public StressTest {
         // no iterator should finish before the first one
         assert(iters[i]->Valid() &&
                iters[i]->key().starts_with(prefix_slices[i]));
+
+        if (ro_copies[i].allow_unprepared_value) {
+          if (!iters[i]->PrepareValue()) {
+            fprintf(stderr,
```

Comment:
Interesting .. I didn't realize in batched_ops_stress we just print to stderr instead of crashing ... how did we even surface the error?

Reply:
I believe any output written to stderr will be caught and reported as error by the crash test script.